api/v1/swagger.yaml

```diff
@@ -3094,6 +3094,17 @@ paths:
             Filter by parentAbsolutePath; asterisk (*) wildcards ok.
           required: false
           type: string
+        -
+          name: tags
+          in: query
+          items:
+            type: string
+            maxLength: 60
```

Comment:
Why 60 here? (Might need to allow for longer since you could build up a list of includes or excludes.)